_posts/en/newsletters/2020-03-11-newsletter.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Bitcoin Optech Newsletter #88'
+permalink: /en/newsletters/2020/03/11/
+name: 2020-03-11-newsletter
+slug: 2020-03-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a description of methods for preventing
+hardware wallets from leaking private information through transaction
+signatures and provides an update on the BIP322 generic signmessage
+protocol.  Also included are our regular sections about new releases and
+notable merges of popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
```

Comment:
(As an aside, it's unclear to me why "Action Items" is now generally empty and "Releases" is down toward the end. Releases and RCs seem like action items to me -- what items should go here now?)

Reply:
If you go back to [newsletter #0](https://bitcoinops.org/en/newsletters/2018/06/08/), you'll see the name of the section was originally "Key action items".  This reflected my desire for the section to be critical information that we wanted people to act on *now*, like security updates.  In later newsletters, we shortened the name to just "action items" and started including things in it that even well-intentioned people put off, like testing RCs or upgrading to the latest maintenance release.[1]  I think that the more we tell people to do stuff they don't do, the less likely they are to pay attention to any really important information we put in that section.  So, for the sake of emphasizing future announcements, I moved the RC and release announcements to a new section; I'll also be pushing back a little bit harder on using the action items section for anything that's not really important.

I did briefly discuss this with some of the guys in private before making the RC/Release change, but I should've mentioned my rationale in public as part of implementing that change so we could've discussed it there.  Sorry.  We can certainly change the format again if you feel strongly about it.

[1] I myself am very often guilty of not testing and running old versions.

Reply:
(I gave this same feedback in irc, but leaving it here too for completeness)

ACK moving *Releases* to their own section for the reasons Dave gives above. I like that we've nudged our readers to test and upgrade by putting new releases in this section, but I think the harm done by giving our readers *Action Item* fatigue outweighs the benefit.